### ISSUE TITLE
Incorrect imported sharp edges

### ISSUE THREAD

**Issue by Pinkyo2386** [Parik27/DragonFF#258]
![image](https://github.com/Parik27/DragonFF/assets/120321868/68fc6752-f608-458e-8240-a9330dfb516c)
When I import this model blender seams to mess up the way the car is shaded, pretty sure its loading the sharp edges incorrectly?
![image](https://github.com/Parik27/DragonFF/assets/120321868/ef5ee948-40c7-44b0-acec-e27fe8f519e9)
![image](https://github.com/Parik27/DragonFF/assets/120321868/cdefbf79-52bb-4d69-97ea-37612480d482)

**Comment by Parik27** [comment]
There was a solution to this in a different thread. Try triangulating the model manually https://github.com/Parik27/DragonFF/issues/248#issuecomment-1738641291

or import without sharp edges.

**Comment by Pinkyo2386** [comment]
> There was a solution to this in a different thread. Try triangulating the model manually [#248 (comment)](https://github.com/Parik27/DragonFF/issues/248#issuecomment-1738641291)
> 
> or import without sharp edges.

I've tried all the import and export methods and they all end with a model that shades differently to the original, ive also tried manually selecting all edges and clearing sharp lines, which gives a similar result of no harsh lighting but everything is smooth and some things aren't shaded correctly like around the door handle, I can try and show more screenshots.
I also saw that other thread but the model is already triangulated so I shouldn't need to triangulate right?
Also its a car mod by Automan. https://www.gtainside.com/en/sanandreas/cars/161896-gta-v-dinka-blista-kanjo/

**Comment by Parik27** [comment]
Well if you import without the "Use Edge Split" option, it shouldn't do any deduplication of the vertices that normally causes these normals import issues as the vertices are imported and exported as-is. 

I'll look into the triangulation solution that the person from the issue mentioned. You are right, it shouldn't matter if the model is already triangulated.

**Comment by Pinkyo2386** [comment]
Yeah I've tried using the Use Edge Split, this is what I get, it removes all the sharp edges but i pretty much get the same result from what I can see
![nosplitswire](https://github.com/Parik27/DragonFF/assets/120321868/b3642539-4f29-472d-8e46-d0b44dc99a16)
![nosplits](https://github.com/Parik27/DragonFF/assets/120321868/dd13ed11-810b-4b86-af7b-cac24db42042)
![splits](https://github.com/Parik27/DragonFF/assets/120321868/bf0cef79-d86a-403f-bb75-e87492b769f9)

**Comment by Pinkyo2386** [comment]
If I import it with split edges and then select all and clear sharp edges I get this, pretty sure its not the right way to do it because certain things need sharp edges and things like the door handle shade weird, but its the closest ive got to shaded correctly
![Screenshot 2023-11-18 133035](https://github.com/Parik27/DragonFF/assets/120321868/d1f46b54-f0b1-46c7-a00e-2b87cd4c6b62)
![Screenshot 2023-11-18 133308](https://github.com/Parik27/DragonFF/assets/120321868/405079f6-469a-4fa3-897b-78dbc085c13f)

**Comment by HughJarze** [comment]
I'm sure that I am running into the same issue as this but im not sure. I have very little knowledge in modelling. When you import the Moonbeam (just the standard moonbeam.dff) on the bonnet the underside texture is doublesided and showing through to the top and there are some issues with the windscreen and bonnet normals.  

<img width="1280" alt="image" src="https://github.com/Parik27/DragonFF/assets/87533936/6d8e0fc4-83c1-4a04-8b21-618115cf1943">
<img width="1280" alt="image" src="https://github.com/Parik27/DragonFF/assets/87533936/62c92f57-cefa-45ce-9e6a-9e656b4c4acf">
<img width="505" alt="image" src="https://github.com/Parik27/DragonFF/assets/87533936/22591fe0-f979-4885-8754-bf9ac56fb7d7">

Standard Moonbeam___________________________Imported and Exported Moonbeam
Any help would be greatly appreciated.

**Comment by Pinkyo2386** [comment]
> I'm sure that I am running into the same issue as this but im not sure. I have very little knowledge in modelling. When you import the Moonbeam (just the standard moonbeam.dff) on the bonnet the underside texture is doublesided and showing through to the top and there are some issues with the windscreen and bonnet normals.
> 
> <img alt="image" width="1280" src="https://user-images.githubusercontent.com/87533936/284082897-6d8e0fc4-83c1-4a04-8b21-618115cf1943.png"> <img alt="image" width="1280" src="https://user-images.githubusercontent.com/87533936/284082915-62c92f57-cefa-45ce-9e6a-9e656b4c4acf.png"> <img alt="image" width="505" src="https://user-images.githubusercontent.com/87533936/284083534-22591fe0-f979-4885-8754-bf9ac56fb7d7.png">
> Standard Moonbeam___________________________Imported and Exported Moonbeam Any help would be greatly appreciated.

weird im not sure what that is but I haven't really had issues with standard SA cars yet, might try the moonbeam, maybe post a screenshot in edit mode, if it has weird blue lines like mine it could be the same maybe

**Comment by keremcodes** [comment]
I have the same problem in GTA:VC. I import and export the model without changing anything and smooth faces turns into sharp faces. I tried to smooth it again in Blender (Face -> Shade Smooth) but no progress.
